core/application/tests/technicalImpl/dbModels/UserModel.test.ts

```diff
@@ -0,0 +1,54 @@
+import UserModel, { UserFields } from '@application/technicalImpl/dbModels/UserModel'
+import { UserDTO } from '@commons/dto/UserDTO'
+import { mockUserWithStringId, mockUserWithNumberId, expectedUser } from '@application/tests/mocks/mockUserData'
+
+describe('UserModel Unit Tests', () => {
+  let userModel: UserModel
```

Comment:
?

Reply:
I have tried to fix all the related issues I noticed. [Commit](https://github.com/CraftsmenLtd/BloodConnect/pull/113/commits/4939bbec6f55a26a9b0bce54078fe226a20ee153)